NEWS.txt

```diff
@@ -1,3 +1,28 @@
+## 10.1.0 (March XX, 2018)
+
+* Replaced isomorphic-fetch with axios.
+
+  Isomorphic-fetch was deprecated around end of 2015, and many dependencies have
```

Comment:
I'd remove mentioning that is was deprecated around 2015. Maybe we can just explain that we changed it because it was deprecated, but not so long ago ? I'm not sure how specific has this explanation to be